<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainLoginFrame</class>
 <widget class="QFrame" name="mainLoginFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>38</number>
     </property>
     <property name="rightMargin">
      <number>27</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="goBackButton">
       <property name="statusTip">
        <string>Go Back</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>left_arrow.png</normaloff>left_arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="loginInputFrame">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>200</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item row="1" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>30</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Password</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="usernameLabel">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>33</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Username</string>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="passwordLabel">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>33</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Password</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>30</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Username</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="errorsLabel">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff1111;&quot;&gt;Errors Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>85</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="submitLoginButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Carlito</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="statusTip">
        <string>Log In</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>116</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>goBackButton</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>submitLoginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
